### PR TITLE
XW-1496 | remove not used styles

### PR DESCRIPTION
**Pull request by bognix**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1496

## Reviewers

@Wikia/x-wing

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: front/main/app/styles/component/index.scss [hunk 1]
```diff
@@ -10,38 +8,17 @@
...
-@import 'discussion-dialog';
-@import 'discussion-open-graph';
-@import 'user-avatar';
-@import 'discussion-contributors';
-@import 'discussion-community-badge';
-@import 'discussion-side-button';
-@import 'discussion-app-promotion';
-@import 'discussion-empty-forum-message';
 @import 'context-links';
 @import 'pop-over';
```
Comment (rogatty) [inline front/main/app/styles/component/index.scss]: This one can be removed too.

File: front/main/app/styles/component/index.scss [hunk 2]
```diff
@@ -55,8 +32,6 @@
 // wikia ui reusable components
 @import 'wikia-ui-components/check-box';
```
Comment (rogatty) [inline front/main/app/styles/component/index.scss]: This one can be removed too.

**Comment by rogatty** [comment]
Two more things to remove and will LGTM

**Comment by tomasznapieralski** [comment]
LGTM tested on both android and safari

**Pull request merged**